$OpenBSD: patch-configure,v 1.6 2010/05/13 15:31:08 ajacoutot Exp $

uuid_generate(3):
    needs e2fs-uuid from sysutils/e2fsprogs over libuuid (devel/uuid).

--- configure.orig	Thu May 13 16:30:01 2010
+++ configure	Thu May 13 16:59:49 2010
@@ -17062,12 +17062,12 @@ if test -n "$PKG_CONFIG"; then
         pkg_cv_UUID_CFLAGS="$UUID_CFLAGS"
     else
         if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"uuid\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "uuid") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"e2fs-uuid\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "e2fs-uuid") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_UUID_CFLAGS=`$PKG_CONFIG --cflags "uuid" 2>/dev/null`
+  pkg_cv_UUID_CFLAGS=`$PKG_CONFIG --cflags "e2fs-uuid" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -17080,12 +17080,12 @@ if test -n "$PKG_CONFIG"; then
         pkg_cv_UUID_LIBS="$UUID_LIBS"
     else
         if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"uuid\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "uuid") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"e2fs-uuid\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "e2fs-uuid") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_UUID_LIBS=`$PKG_CONFIG --libs "uuid" 2>/dev/null`
+  pkg_cv_UUID_LIBS=`$PKG_CONFIG --libs "e2fs-uuid" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -17104,14 +17104,14 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        UUID_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "uuid"`
+	        UUID_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "e2fs-uuid"`
         else
-	        UUID_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "uuid"`
+	        UUID_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "e2fs-uuid"`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$UUID_PKG_ERRORS" >&5
 
-	as_fn_error "Package requirements (uuid) were not met:
+	as_fn_error "Package requirements (e2fs-uuid) were not met:
 
 $UUID_PKG_ERRORS
 
@@ -17369,7 +17369,7 @@ if test "${ac_cv_lib_id3_ID3Tag_New+set}" = set; then 
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lid3 -lstdc++ -lz $LIBS"
+LIBS="-lid3 -lstdc++ -lz -lm $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -17404,7 +17404,7 @@ if test "x$ac_cv_lib_id3_ID3Tag_New" = x""yes; then :
 #define HAVE_LIBID3 1
 _ACEOF
 
-  LIBS="-lid3 $LIBS"
+  LIBS="-lid3 -lm $LIBS"
 
 fi
 
